src/stores/ContractsStore.js

```diff
@@ -33,15 +33,18 @@ class ContractsStore {
 	@observable keysBallotThreshold;
 	@observable minThresholdBallotThreshold;
 	@observable proxyBallotThreshold;
+	@observable validatorLimits;
```

Comment:
Duplicate of https://github.com/poanetwork/poa-dapps-voting/pull/50/files#diff-795e7edef556435c55ff76927dc5369eR22

Reply:
👍